inference-engine/src/mkldnn_plugin/mkldnn_memory.cpp

```diff
@@ -101,13 +101,13 @@ void MKLDNNMemory::Create(MemoryDescPtr desc, const void* data, bool pads_zeroin
     }
 
     if (pMemDesc->isDefined()) {
-        Create(MemoryDescUtils::convertToDnnlMemoryDesc(*pMemDesc)->getMklDesc(), data, pads_zeroing);
+        Create(MemoryDescUtils::convertToDnnlMemoryDesc(*pMemDesc)->getDnnlDesc(), data, pads_zeroing);
     } else {
         //delayed dynamic allocation
         size_t maxMemSize = pMemDesc->getMaxMemSize();
         VectorDims dummySize{MemoryDesc::UNDEFINED_SIZE == maxMemSize ? 1 : maxMemSize};
         DnnlBlockedMemoryDesc dummyDesc(InferenceEngine::Precision::U8, Shape(dummySize));
-        Create(dummyDesc.getMklDesc(), data, false);  // no pads zeroing
+        Create(dummyDesc.getDnnlDesc(), data, false);  // no pads zeroing
```

Comment:
I will do full check in https://github.com/maxnick/openvino/pull/23 because in this PR I will need do some fixes to infer dynamic network.

At this moment I checked that
`DnnlBlockedMemoryDesc dummyDesc(InferenceEngine::Precision::U8, Shape(dummySize));

 Create(dummyDesc.getDnnlDesc(), data, false);  // no pads zeroing`
 does not throw an exception